bin/compile

```diff
@@ -273,6 +275,13 @@ sub_env "$BIN_DIR/steps/geo-libs"
 # shellcheck source=bin/steps/gdal
 source "$BIN_DIR/steps/gdal"
 
+# SQLite3 support.
+# This sets up and installs sqlite3 dev headers and the sqlite3 binary
```

Comment:
Is it worth mentioning here or in `bin/steps/sqlite3` that:
* `libsqlite3-0` isn't being installed here since it already exists in the stack images
* the reason the headers aren't installed in the stack image is https://github.com/heroku/heroku-buildpack-python/issues/427#issuecomment-316794743